src/encoded/audit/file.py

```diff
@@ -302,57 +322,13 @@ def audit_file_controlled_by(value, system):
                                    detail, level='WARNING')
                 return
 
+# def audit_file_chip_seq_control_read_depth(value, system):
+# migrated to experiment https://encodedcc.atlassian.net/browse/ENCD-3493
+
 # def audit_file_flowcells(value, system): # removed in version 56
 # http://redmine.encodedcc.org/issues/5060
 
 
-@audit_checker('File', frame=['paired_with'])
-def audit_paired_with(value, system):
-    '''
-    A file with a paired_end needs a paired_with.
-    Should be handled in the schema.
-    A paired_with should be the same replicate
-    '''
-
-    if 'paired_end' not in value:
-        return
-
-    if value['paired_end'] in ['1,2']:
-        return
-
-    if 'paired_with' not in value:
-        return
-
-    if 'replicate' not in value['paired_with']:
-        return
-
-    if 'replicate' not in value:
-        detail = 'File {} has paired_end = {}. It requires a replicate'.format(
-            value['@id'],
-            value['paired_end'])
-        yield AuditFailure('missing replicate', detail, level='INTERNAL_ACTION')
-        return
-
-    if value['replicate'] != value['paired_with']['replicate']:
-        detail = 'File {} has replicate {}. It is paired_with file {} with replicate {}'.format(
-            value['@id'],
-            value.get('replicate'),
-            value['paired_with']['@id'],
-            value['paired_with'].get('replicate'))
-        yield AuditFailure('inconsistent paired_with', detail, level='ERROR')
-        return
-
-    if value['paired_end'] == '1':
-        context = system['context']
-        paired_with = context.get_rev_links('paired_with')
-        if len(paired_with) > 1:
-            detail = 'Paired end 1 file {} paired_with by multiple paired end 2 files: {!r}'.format(
-                value['@id'],
-                paired_with,
-            )
-            yield AuditFailure('multiple paired_with', detail, level='ERROR')
-            return
-
 '''
```

Comment:
Can we remove this big comment?  It has it's own audit_checker/frame so it's not even compatible.
You could leave a bread crumb to a branch/tag/jira issue that has it incase it needed to be ressurected.

Reply:
DONE